uaclient/gpg.py

```diff
@@ -1,55 +1,24 @@
 import logging
 import os
+import shutil
 
 from uaclient import exceptions
-from uaclient import util
 
 
-def export_gpg_key_from_keyring(
-    key_id: str, source_keyring_file: str, destination_keyfile: str
-) -> None:
-    """Export a specific key from source_keyring_file into destination_keyfile
+def export_gpg_key(source_keyfile: str, destination_keyfile: str) -> None:
+    """Copy a specific key from source_keyring_dir into destination_keyfile
 
-    :param key_id: Long fingerprint of key to export.
-    :param source_keyring_file: The keyring file from which to export.
+    :param source_keyfile: Path of source keyring file to export.
     :param destination_keyfile: The filename created with the single exported
         key.
 
     :raise UserFacingError: Any GPG errors or if specific key does not exist in
         the source_keyring_file.
     """
-    export_cmd = [
-        "gpg",
-        "--output",
-        destination_keyfile,
-        "--yes",
-        "--no-auto-check-trustdb",
-        "--no-options",
-        "--no-default-keyring",
-        "--keyring",
-        source_keyring_file,
-        "--export",
-        key_id,
-    ]
-    logging.debug("Exporting GPG key %s from %s", key_id, source_keyring_file)
-    try:
-        out, err = util.subp(export_cmd)
-    except util.ProcessExecutionError as exc:
-        with util.disable_log_to_console():
-            logging.error(str(exc))
+    logging.debug("Exporting GPG key %s", source_keyfile)
```

Comment:
This log message, and the function name, aren't really accurate any longer. We aren't exporting a key, we're installing a keyring.